README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
+of executing that mission.
+
+Since the Foundation and the TSC are intended to support the community the size and scope
+of the work and projects in it will grow to accomodate the needs of the community.
+
```

Comment:
> Even without us having collected a full list, I would assume that resources are not infinite :) I could guess: CI resources, marketing dollars, TSC staff time. I think we will learn this more clearly as we try to grow the scope.

We'll certainly continue to identify more areas where we are limited and can taper accepting applications in accordance with those resources. However, all the things you listed aren't even entitlements projects receive :)

**CI resources:** This is under the CTC and core. The current Build WG is *for* core and those resource aren't allocated to incoming projects. Additional projects will have their own resources and, if they need more, will build up their own competency similar to the CTC Build WG.

**Markting dollars:** The foundation doesn't market *core* they market Node.js and that already includes relevant projects written in Node.js inside or outside of the foundation whenever relevant. In fact, having more projects under the same roof only helps them bring in the right people to help with that messaging when necessary.

**TSC staff time:** The TSC is not allocated to incoming projects, incoming projects are given a *mentor(s)* which isn't necessarily a TSC member, the list of mentors is on the README and is currently only a single person (me). This will grow over time but the reason it is separate from the TSC is so that we can grow the list of members wider than the TSC membership. Other than that the only thing the TSC will be spending time on is evaluating applications and eventually charters once projects are ready to graduate the incubator.

Every project is different, it's possible that some projects will have needs we haven't even considered or identified yet. The only productive step I can see us taking is adding an "Infrastructure" section to applications and working closely with applicants to flush it out and identify all the necessary resource allocations, if any, but we shouldn't be surprised if there is little if nothing they need other than a proper home.

 > How about inviting representative of potential member projects to the TSC meetings for some time, so we have a chance to learn from the perspectives of the community?

There's two problems with this. The first is that the TSC membership has a narrow view of the community as a whole. None of us really understand how wide the community is and could invite the right people. The second problem is that we are unlikely to find the areas of the community *in need* of our support this way. Some projects already appear to need some support from the foundation while others seem perfectly fine without us, but that is *our* interpreation of what is going on and the reality may be very different. The only way we can respond to what the community actually needs is to let them tell us what they need by applying.